### ISSUE TITLE
[Request] Line marker for policies and models

### ISSUE THREAD

**Issue by olivernybroe** [laravel-idea/plugin#90]
Recently a new line marker was added in for the relation between events and event listener, and today I ran into another place where line markers would be really helpful.

# Description 
So in Laravel we have policies. Policies are linked to a model. 
Adding a line marker for each model which navigates to the policy and vice versa would be really awesome.

## Registration of policies
Policies can be registered in three ways, two of them which should be fairly straightforward to implement.
In the `AuthServiceProvider`, we have an array called `policies`, this array is a key values store between models and policies.
```php
    protected $policies = [
        Post::class => PostPolicy::class,
    ];
```
There is also an auto discovery which is just removing the suffix `Policy` and checks if a model exist with the name.

The last method, which I don't think should be supported by this plugin, is you can define a custom method which is used to guess the name.

## Current solution
My current solution is adding a docblock comment with the `@see` tag and link the class, as phpstorm understands that.
```php
/**
 * Contains all the policies for files.
 *
 * @see File
 */
class FilePolicy
```

Documentation: https://laravel.com/docs/7.x/authorization#registering-policies

**Comment by adelf** [comment]
Thanks. Will be added to the next version.

**Comment by adelf** [comment]
Implemented in the 3.2 version. Please write here if I missed something.
